### PR TITLE
Fix button name errors in e2e test

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

**Comment by lbodor** [comment]
What other errors does the second commit fix? The old-standing "alert" error?

**Comment by frankfu03** [comment]
Yes, we still have "alert" errors plus timeout errors. I have dropped the 2nd commit and put it to a separate PR.

File: src/client/app/site-information/site-location.component.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
         </span>
         <span [hidden]="siteLocation != null" class="pull-right">
             <button class="btn btn-similar btn-xs"
+                    id="new-{{getElementName()}}"
                     title="Add a new {{getItemName()}}"
```
Comment (lbodor) [inline src/client/app/site-information/site-location.component.html]: Do you think that we could at some stage remove new and delete buttons from site location?

Comment (frankfu03) [reply src/client/app/site-information/site-location.component.html]: Yes.

**Pull request merged**